mlir/cmake/modules/AddMLIRPython.cmake

```diff
@@ -228,22 +228,27 @@ endfunction()
 #     aggregate dylib that is linked against.
 function(declare_mlir_python_extension name)
   cmake_parse_arguments(ARG
-    ""
-    "ROOT_DIR;MODULE_NAME;ADD_TO_PARENT"
+    "SUPPORT_LIB"
+    "ROOT_DIR;MODULE_NAME;ADD_TO_PARENT;SOURCES_TYPE"
```

Comment:
Do we need to list `SOURCES_TYPE` here? I don't see `ARG_SOURCES_TYPE` used anywhere...

Reply:
Whoops (left over from when I considered exposing SOURCES_TYPE directly instead of SUPPORT_LIB

Reply:
done